torch/_inductor/cpp_builder.py

```diff
@@ -538,6 +538,9 @@ def _get_optimization_cflags(cpp_compiler: str) -> List[str]:
             cflags.append("fno-unsafe-math-optimizations")
         if not config.cpp.enable_floating_point_contract_flag:
             cflags.append("ffp-contract=off")
+        if config.cpp.enable_floating_point_contract_fast_flag:
```

Comment:
Similar to my last comment,  I think now we can have both "ffp-contract=off" and "ffp-contract=fast" in the compile flags.